vllm-tt-metal-llama3-70b/src/run_vllm_api_server.py

```diff
@@ -30,11 +27,22 @@ def get_encoded_api_key(jwt_secret):
     return encoded_jwt
 
 
-@patch.object(LLMEngine, "__init__", new=logging_init_wrapper)
 def main():
+    # set up logging
+    config_path, log_path = set_vllm_logging_config(level="DEBUG")
+    print(f"setting vllm logging config at: {config_path}")
+    print(f"setting vllm logging file at: {log_path}")
+    # note: the vLLM logging environment variables do not cause the configuration
+    # to be loaded in all cases, so it is loaded manually in set_vllm_logging_config
+    os.environ["VLLM_CONFIGURE_LOGGING"] = "1"
+    os.environ["VLLM_LOGGING_CONFIG"] = str(config_path)
+    # stop timeout during long sequential prefill batches
+    # e.g. 32x 2048 token prefills taking longer than default 30s timeout
+    # timeout is 3x VLLM_RPC_TIMEOUT
+    os.environ["VLLM_RPC_TIMEOUT"] = "200000"  # 200000ms = 200s
```

Comment:
Does it make sense to put this in the `.env` file that gets passed during the launch? Along with the other variables such as `WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml`